packages/contracts-bedrock/.gas-snapshot

```diff
@@ -8,4 +8,4 @@ GasBenchMark_L1StandardBridge_Finalize:test_finalizeETHWithdrawal_benchmark() (g
 GasBenchMark_L2OutputOracle:test_proposeL2Output_benchmark() (gas: 92951)
 GasBenchMark_OptimismPortal:test_depositTransaction_benchmark() (gas: 68360)
 GasBenchMark_OptimismPortal:test_depositTransaction_benchmark_1() (gas: 68991)
-GasBenchMark_OptimismPortal:test_proveWithdrawalTransaction_benchmark() (gas: 155557)
\ No newline at end of file
+GasBenchMark_OptimismPortal:test_proveWithdrawalTransaction_benchmark() (gas: 155553)
```

Comment:
I don't understand why, but adding [these vm.labels](https://github.com/ethereum-optimism/optimism/pull/9317/files#diff-91b2122803f6a5d39a9c95ef8c90ee82ef8d14647ddb31617b33cd312237f943R179-R189) caused this to use less gas